Composer/packages/client/src/pages/publish/BotStatusList.tsx

```diff
@@ -16,14 +16,56 @@ import { ActionButton, IconButton } from 'office-ui-fabric-react/lib/Button';
 import { SharedColors } from '@uifabric/fluent-theme';
 import { FontSizes } from '@uifabric/styling';
 import get from 'lodash/get';
-import { ITextField, TextField } from 'office-ui-fabric-react/lib/TextField';
+import { useCopyToClipboard } from '@bfc/ui-shared';
+import { Callout } from 'office-ui-fabric-react/lib/Callout';
 
 import { ApiStatus } from '../../utils/publishStatusPollingUpdater';
 
 import { PublishStatusList } from './PublishStatusList';
 import { detailList, listRoot, tableView } from './styles';
 import { BotPublishHistory, BotStatus } from './type';
 
+const copiedCalloutStyles = {
+  root: {
+    padding: '10px',
+  },
+};
+
+type SkillManifestUrlFieldProps = {
+  url: string;
+};
+
+const SkillManifestUrlField = ({ url }: SkillManifestUrlFieldProps) => {
+  const { isCopiedToClipboard, copyTextToClipboard, resetIsCopiedToClipboard } = useCopyToClipboard(url);
+
+  const calloutTarget = useRef<EventTarget>();
```

Comment:
change this to HTMLElement to fix the build

Reply:
Thank you!